packages/rancher-gatekeeper/generated-changes/patch/crds/config-customresourcedefinition.yaml.patch

```diff
@@ -1,11 +0,0 @@
---- charts-original/crds/config-customresourcedefinition.yaml
```

Comment:
I can't really follow why these patches were excluded; were these patches included now in upstream?